pkg/nsx/services/vpc/builder.go

```diff
@@ -21,7 +21,7 @@ func generateIPBlockKey(block model.IpAddressBlock) string {
 	cidr := block.Cidr
 	vpc_uid := ""
 	for _, tag := range block.Tags {
-		if *tag.Scope == common.TagScopeVPCCRUID {
+		if *tag.Scope == common.TagScopeNetworkInfoCRUID {
```

Comment:
Also for IPBlock, I don't think we should add networkinfo CR in tag

Reply:
fixed